src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -125,6 +128,7 @@ const convertInclude = async (includeBlock, pageData, ...restParams) => {
       href,
       index,
       type: includeType,
+      showEmbedError,
```

Comment:
Rather than referencing the name of a front-end action and/or component here, can we call this something like `hasAmpSupport`? Consider it a bit like the `enabled` flag - it's up to the consumer what it wants to do when this value is `true` or `false`.

Reply:
I think I named it `hasNoAmpSupport` because my thought process was when the front end container checked the boolean, it being `true` would show the placeholder. Now thinking about it `hasAmpSupport` is better as we just have to negate the check to show the placeholder, and now we have the added benefit of `hasAmpSupport` if we need its use in the future.